src/redux/bids/index.ts

```diff
@@ -107,7 +105,10 @@ const keyFinder = (object: BidsBySaleId) => {
   return String(Object.getOwnPropertyNames(object)[0])
 }
 
-const eventExists = (events: SaleBid[], event: SaleBid[]) => {
+const eventExists = (
```

Comment:
Could you elaborate on this, please?

Reply:
checks for purchases with the same id between payload and state

Reply:
a function that checks if a purchase event in the action payload is already in the redux state. This is to prevent duplicates from occurring as there are two sources of information in the interface; one that reads directly from the chain and one from the subgraph. 

I assume that there apollo has a better way of dealing with duplicates and caching.